tests/_core/nodes.yaml

```diff
@@ -7,3 +7,53 @@ chapters:
     method: GET
     response:
       status: 200
+  - synopsis: Nodes thread_pool metric with a json response.
+    path: /_nodes/{metric}
+    method: GET
+    parameters:
+        metric:
+          - thread_pool
+    response:
+      status: 200
+  - synopsis: Nodes jvm and thread_pool metric with a json response.
+    path: /_nodes/{metric}
+    method: GET
+    parameters:
+      metric:
+        - jvm
+        - thread_pool
+    response:
+      status: 200
+  - synopsis: Nodes _all metric with a json response.
+    path: /_nodes/{metric}
+    method: GET
+    parameters:
+      metric:
+        - _all
+    response:
+      status: 200
+  - synopsis: Nodes _cluster_manager metric with a json response.
+    path: /_nodes/{metric}/stats
+    method: GET
+    parameters:
+      metric:
+        - _cluster_manager
+    response:
+      status: 200
+  - synopsis: Nodes _cluster_manager metric with a json response.
+    path: /_nodes/{metric}/stats
+    method: GET
+    parameters:
+      metric:
+        - data:true
```

Comment:
@dblock do you have any idea how to implement a fix for this test ?

Reply:
This error?

```
FAILED  Nodes _cluster_manager metric with a json response. 
            FAILED  PARAMETERS 
                FAILED  metric (data/0 must be equal to one of the allowed values)
```

The `data:true` value doesn't seem like a valid metric.

Reply:
it's seems weird yes, but it's define in documentation

Reply:
Is it read like a JSON value? Quote it, `"data:true"`?

Reply:
if I do that, that implies that I have to put it in the enum and therefore do I have to do the same for `data:false` and the other filters?

Reply:
I suppose that yes? There's really no such thing as booleans in URL paths AFAIK, this is a very creative use of the API.

Reply:
These should be using `/_nodes/{node_id}/stats`